swift/ios-shoppe-demo/ios-shoppe-demo.xcodeproj/project.pbxproj

```diff
@@ -15,13 +15,19 @@
 		F25D20A9240EB5A600CF89C3 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = F25D20A7240EB5A600CF89C3 /* LaunchScreen.storyboard */; };
 		F25D20B4240EB5A600CF89C3 /* ios_shoppe_demoTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = F25D20B3240EB5A600CF89C3 /* ios_shoppe_demoTests.swift */; };
 		F25D20BF240EB5A600CF89C3 /* ios_shoppe_demoUITests.swift in Sources */ = {isa = PBXBuildFile; fileRef = F25D20BE240EB5A600CF89C3 /* ios_shoppe_demoUITests.swift */; };
+<<<<<<< HEAD:swift/ios-shoppe-demo/ios-shoppe-demo.xcodeproj/project.pbxproj
```

Comment:
uh-oh! I think you committed a merge conflict @HaroldFS

Reply:
Its good to go now, I merged master into this branch and I fixed my conflicts of directory pathMaster already has the ShoppeItem however it it is in a new location since I changed the Directory name to reflect Sabrina's suggestion.